diff --git a/wrangler-dist/cli.js b/wrangler-dist/cli.js
index 461ca04..5bda77c 100644
--- a/wrangler-dist/cli.js
+++ b/wrangler-dist/cli.js
@@ -124361,7 +124361,7 @@ async function buildImage(dockerPath, options) {
     pathToDockerfile: options.dockerfile,
     buildContext: options.image_build_context,
     args: options.image_vars,
-    platform: "linux/amd64"
+    platform: undefined
   });
   return dockerBuild(dockerPath, { buildCmd, dockerfile });
 }
@@ -124745,7 +124745,7 @@ function getEgressInterceptorImage() {
 }
 async function pullEgressInterceptorImage(dockerPath) {
   const image = getEgressInterceptorImage();
-  await runDockerCmd(dockerPath, ["pull", image, "--platform", "linux/amd64"]);
+  await runDockerCmd(dockerPath, ["pull", image]);
 }
 async function pullImage(dockerPath, options, logger5) {
   const domain3 = new URL(`http://${options.image_uri}`).hostname;
@@ -124764,8 +124764,8 @@ async function pullImage(dockerPath, options, logger5) {
     "pull",
     options.image_uri,
     // All containers running on our platform need to be built for amd64 architecture, but by default docker pull seems to look for an image matching the host system, so we need to specify this here
-    "--platform",
-    "linux/amd64"
+    // "--platform",
+    // "linux/amd64"
   ]);
   const ready = pull.ready.then(async ({ aborted: aborted2 }) => {
     if (!aborted2) {
